gettsim/demographic_vars.py

```diff
@@ -327,3 +327,81 @@ def rentner_in_hh(hh_id: IntSeries, rentner: BoolSeries) -> BoolSeries:
     BoolSeries indicating pensioner in household.
     """
     return rentner.groupby(hh_id).any()
+
+
+def steuerklasse(
```

Comment:
I think it might make sense to move this function to `lohn_st.py`.

Reply:
OK